HaaLimits/python/HaaLimits2DNew.py

```diff
@@ -312,7 +312,10 @@ def fitSignals(self,h,region='PP',shift='',**kwargs):
             results[h] = {}
             errors[h] = {}
             integrals[h] = {}
-        if self.YRANGE[1] > 100: initialValuesDCB = self.GetInitialValuesDCB(isKinFit=isKinFit)
+        if self.YRANGE[1] > 100: 
+            if yFitFunc == "DCB_Fix": initialMeans = self.GetInitialDCBMean()
```

Comment:
DCB = typical DCB fitting
DCB_Fix = the same except additionally fixing the mean. It was one of the test I did to try to improve the DCB. Don't use it, since it didn't have much effect. I just left it in there in case we wanted to do more testing with it.  

You should use:
yFitFuncFP = "DCB"
yFitFuncPP = "DG"